gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-bin.zip
-distributionSha256Sum=b49c6da1b2cb67a0caf6c7480630b51c70a11ca2016ff2f555eaeda863143a29
```

Comment:
Why checksum verification was removed?

Reply:
Indeed, good point

Reply:
Thanks for reminding! Forgot to add it to the Wrapper task configuration.

Reply:
I was trying to find that in the docs but I failed so I'm gonna ask here: why are some properties left in `gradle-wrapper.properties` and you're moving only distribution related properties into `build.gradle` ? What's the advantage / can all be moved there?
I'm guessing this is because of this: https://stackoverflow.com/questions/25769536/how-when-to-generate-gradle-wrapper-files#comment84284279_38113324 but I'm just wondering if we can get rid of this file completely.

Reply:
That is good point! With wrapper task it's possible to configure everything:
https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/wrapper/Wrapper.html

One advantage of having the `properties` and `wrapper.jar` file in the repo, is that in order to run a build some do not need to have Gradle pre-installed on the system. Only Java would be required. Also there would not be any problems if older version of Gradle is installed locally. In order to run `wrapper` task, Gradle build should pass configuration phase. If version of Gradle installed on host doesn't match version which is supported by project, then it becomes not as easy to configure Wrapper. 
Otherwise wrapper can be generated by running `gradle wrapper` (not `./gradlew`) from the root of this repository on host with pre-installed Gradle.